doc/integrator/docker.rst

```diff
@@ -66,12 +66,23 @@ Otherwise:
    docker images | grep "<none>" | awk '{print $3}' | xargs --no-run-if-empty docker rmi || true
    docker volume ls --quiet --filter dangling=true | grep '[0-9a-f]\{64\}' | xargs --no-run-if-empty docker volume rm
 
-This will remove::
+This will remove:
```

Comment:
Are you sure about this change?

Reply:
Yes, else the following bullet points are not interpreted and displayed as a code section.
Here is how the page looks like after my changes:
![Screenshot from 2020-02-17 11-13-38](https://user-images.githubusercontent.com/1192331/74644477-9f5ec380-5176-11ea-898c-ff5c1325b596.png)

Reply:
+1 thanks :-)